src/lists/list-item.jsx

```diff
@@ -296,16 +296,18 @@ const ListItem = React.createClass({
   },
 
   _createTextElement(styles, data, key) {
-    const isAnElement = React.isValidElement(data);
-    const mergedStyles = isAnElement ?
-      Object.assign({}, styles, data.props.style) : null;
-
-    return isAnElement ? (
-      React.cloneElement(data, {
+    if (React.isValidElement(data)) {
+      let style = Object.assign({}, styles, data.props.style);
+      if (typeof data.type === 'string') { // if element is a native dom node
```

Comment:
I guess in my opinion, if a user wanted to have prepared styles in their own components, they should use and call the prepareStyles function from muiTheme themselves applying it to a dom node.

If they were using a home baked component that had a material-ui component inside of it and passed the prepared style to the material-ui component, it could break. I think as a rule we shouldn't pass prepared styles to anything but dom nodes.